### ISSUE TITLE
Docker image fails on Raspberry Pi

### ISSUE THREAD

**Issue by chacal** [openthread/borderrouter#224]
**Describe the bug**
Running the latest openthread/otbr image from Docker Hub on Raspberry Pi 3 fails with error:
`standard_init_linux.go:190: exec user process caused "exec format error"` 

The error message suggests that the image is not build for ARM architecture, I believe.

**To Reproduce**
1. Flash latest Raspbian Stretch Lite on an SD card
1. Boot the image and log in
1. Install Docker: `curl -sSL https://get.docker.com | sh`
1. Try to run otbr image from Docker Hub
```
docker run --sysctl "net.ipv6.conf.all.disable_ipv6=0 \
net.ipv4.conf.all.forwarding=1 net.ipv6.conf.all.forwarding=1" \
-p 8080:80 --dns=127.0.0.1 -it --volume \
/dev/ttyACM0:/dev/ttyACM0 --privileged openthread/otbr \
--ncp-path /dev/ttyACM0
```

**Expected behavior**
OTBR should start normally.

**Console/log output**
Running the image generates an error:

`standard_init_linux.go:190: exec user process caused "exec format error"`

**Comment by Vyrastas** [comment]
Hi @chacal 

You mention Raspbian Stretch Lite... the [guide on openthread.io](https://openthread.io/guides/border-router/docker) requires Raspbian Stretch with Desktop.  Can you try it with that?

**Comment by chacal** [comment]
Hi @Vyrastas!

Tried also with the Desktop image, but the end result is same.

If I build the Docker image by myself on the Raspberry Pi, it seems to work. So the problem seems to be the prebuilt image in Docker Hub (which most probably is only built for x86 architecture and not ARM).

**Comment by Vyrastas** [comment]
Yes, that is likely the issue.  We'll compile a second image on ARM and upload it to the Docker Hub, update the docs, etc.  Thanks for bringing this to our attention, please just use the one you built directly on the Pi in the meantime.

**Comment by Vyrastas** [comment]
@chacal 

I've uploaded an ARM-specific image to the Hub, and updated `openthread/otbr:latest` to be a manifest that will pull the correct image depending on your underlying system (Linux, RPi3B). If you try the `docker pull` again, it should work correctly.

**Comment by chacal** [comment]
Just FYI.. Now everything works just like it should also for me. Thanks! 👍
